src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,87 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.storage.Storage;
+import duke.task.*;
+import duke.ui.Ui;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class AddCommand extends Command {
+    private String type;
+    private String description;
+
+    private static final String MESSAGE_ADD      = "Got it. I've added this task:\n  %s\n"
+            + "Now you have %d %s in the list.";
```

Comment:
Great design thinking on your part to declare all this repetitive messages as constants (static final), and once again good use of format specifiers in those Strings!